main.go

```diff
@@ -167,7 +167,7 @@ func updateRoleMappings() {
 	roleMappingsUpdated := transformRoleMappings(roleMappings, awsIAMRoles, accountId)
 
 	// Add worker node role bindings if those are absent
-	iamRoleARN := getInstanceRole()
+	iamRoleARN := "arn:aws:iam::" + accountId + ":role/"getInstanceRole()
```

Comment:
```suggestion
	iamRoleARN := "arn:aws:iam::" + accountId + ":role/" + getInstanceRole()
```

Reply:
I think using plus signs will add spaces between each component, would it make more sense to use `fmt.Splrintf()`?